manager/src/main/resources/logging-dev.properties

```diff
@@ -28,16 +28,14 @@ java.util.logging.ConsoleHandler.formatter=org.openremote.container.util.LogForm
 .level=INFO
 
 # The root logger category of OpenRemote
-org.openremote.level=FINE
+org.openremote.level=FINEST
```

Comment:
Was this done for debugging your code only or will it help with future debugging as well?
Doesn't it generate a lot of logging causing performance issues while debugging?